src/spikeinterface/sorters/external/kilosort2_5_master.m

```diff
@@ -62,7 +64,7 @@ function kilosort2_5_master(fpath, kilosortPath)
             rez.ops.Nbatch = Nbatch;
             rez.ops.NTbuff = NTbuff;
 
-            tic;  % tocs are coming
+            tic;  % tocs are coming TODO
```

Comment:
The tocs are actually in the KS codebase, see https://github.com/SpikeInterface/spikeinterface/pull/1887